virtual_rainforest/models/animals/protocols.py

```diff
@@ -0,0 +1,32 @@
+"""The `models.animals.protocols` module contains a class provides eatability definition
+used by AnimalCohorts, PlantCommunity, and Carcasses in the
+:mod:`~virtual_rainforest.models.animals` module.
+"""  # noqa: D205, D415
+
+from typing import Protocol
+
+from virtual_rainforest.models.animals.functional_group import FunctionalGroup
+
+
+class Consumer(Protocol):
+    """This is the protocol for defining consumers (currently just AnimalCohort)."""
+
+    intake_rate: float
+    functional_group: FunctionalGroup
+    individuals: int
+
+
+class Pool(Protocol):
+    """This is a protocol for defining dummy abiotic pools containing energy."""
+
+    stored_energy: float
+
+
+class Resource(Protocol):
+    """This is the protocol for defining what classes work as trophic resources."""
+
+    stored_energy: float
```

Comment:
It may end up working that way but it's not clear to me exactly what that is going to look like once things are stoichiometrically explicit. Some of these protocols may be abandoned as toy pools are trashed and animals are drawing on pools in other modules, which are handled in different ways. I will revisit the idea once we are further along.